### DISCUSSION TITLE
RX Failsafe Position when using serial link

### DISCUSSION THREAD

**Discussion by Arielofek** [ExpressLRS/ExpressLRS#2911] [Ideas]
Hello,
In my set up I have an ELRS receiver linked through SBUS to a gyro with Level mode capability. I would like to set the position of the channel that controls the gyro mode to Level mode. However, the position set up of ELRS RX only works for PWM outputs. This forces me to give up the nice single control line of SBUS and go back using the old PWM setup which is less desired because of the additional wire connection. I will be more than happy to see in next ELRS releases an updated that sets the failsafe position in both PWM and SBUS/other serial links.

**Comment by sk8board** [comment]
I asked for this a while ago and was told by a dev that it is planned for ELRS v3.6. https://github.com/ExpressLRS/ExpressLRS/discussions/2514

You could use my project as a short term solution until ELRS v3.6 becomes available.

https://github.com/sk8board/CRServoF_ELRS_for_RP2040_Pico

**Reply by pkendall64** [reply]
Where does it say that a dev said that would be a feature for 3.6? I can't find it anywhere!

**Reply by sk8board** [reply]
Here in the comments.

https://www.youtube.com/watch?v=jxUJyF7HmXE&lc=

Is this not planned for v3.6? If not, when do you expect failsafe settings to become available in non-PWM receivers?

**Reply by sk8board** [reply]
@pkendall64 Paul, did you see the comment that I was referring to?

When do you expect failsafe settings to become available in non-PWM receivers?

**Reply by sk8board** [reply]
@pkendall64 were you contacted by this guy? I think he also goes by the alias of Arielofek.

![image](https://github.com/user-attachments/assets/5b7fe6f6-0a89-48ef-a579-4003834722bd)
